_tutorials/2016-05-22-getting-started-on-xrv64.md

```diff
@@ -0,0 +1,118 @@
+---
+published: false
+date: "2016-05-22 13:49 -0700"
+title: Getting Started on XRv64
+permalink: "https://xrdocs.github.io/application-hosting/tutorials/xrv64-quickstart"
+author: Lisa Roach
+excerpt: "Getting started with Cisco's XRv64 Vagrant Instance."
+tags: 
+  - vagrant
+  - iosxr
+  - cisco
+  - XRv64
+---
+### Quick Start Using XRv64 Box image
+Quick guide to get up and running with XRv64 and vagrant/Virtualbox.
+ 
+Install Virtualbox and vagrant tools to your mac:
+
+Source: http://sourabhbajaj.com/mac-setup/Vagrant/README.html
+
+*	Recommend using the Homebrew package manager.
+
+*	Make sure you install version 5.x virtualbox
+
+*	Vagrant latest version is: 1.8.2
+
+*	Website:  http://brew.sh/
+
+        /usr/bin/ruby -e "$(curl -fsSLhttps://raw.githubusercontent.com/Homebrew/install/master/install)"
+        brew cask install virtualbox
+        brew cask install vagrant
+        brew install socat
+
+Note you may need to install pexpect:
+
+	brew cask install python
+	pip install pexpect
+
+#### Single Node:
+
+	vagrant init xrv64
+  ![vagrant init xrv64](https://xrdocs.github.io/xrdocs-images/assets/images/xrv64_vagrant_init.png)
+   
+	vagrant box add --name xrv64 <REPLACE THIS WITH PUBLIC VAGRANT BOX URL> --force
+    
+  ![vagrant box add](https://xrdocs.github.io/xrdocs-images/assets/images/xrv64_vagrant_add.png)
+  
+	vagrant up 
+
+This will take some time, possibly over 10 minutes once you see the "Waiting for machine to boot" message.  Look for the green "vagrant up" welcome message to confirm the machine has booted:
+	
+   ![vagrant up](https://xrdocs.github.io/xrdocs-images/assets/images/xrv64_vagrant_up_s.png)
+    
+
+
+Now you have two options for accessing the Vagrant instance:
+1. Access the XR Linux shell:
+  		
+       vagrant ssh
+
+  ![vagrant ssh](https://xrdocs.github.io/xrdocs-images/assets/images/xrv64_vagrant_ssh_s.png)
+
+2.  Access XR Console:
+ 
+        ssh -p <forwarded port> vagrant@127.0.0.1
+
+   * Replace `<forwarded port>` with the port forwarded for your 22 port
+
+      * You can check which ports were forwarded with the command:
+
+		    vagrant port
+      
+    * The password is “vagrant” 
+    
+  ![ssh console](https://xrdocs.github.io/xrdocs-images/assets/images/xrv64_ssh_console_s.png)
+
+#### Multi Node:
+
+![Topology](https://xrdocs.github.io/xrdocs-images/assets/images/xrv64_topo_m.png)
+
+Copy the multi-node Vagrantfile: `cp xr-cp-vbox/Vagrantfile`
+
```

Comment:
umm,  You might want to point to to a future github repo (where Rich will put the Vagrantfile).